sdk/typescript/src/signers/txn-data-serializers/local-txn-data-serializer.ts

```diff
@@ -238,7 +238,6 @@ export class LocalTxnDataSerializer implements TxnDataSerializer {
         gasPayment = publish.gasPayment;
         break;
     }
-    // TODO: support batch transaction
```

Comment:
Removing since batch transactions are going away, and programmable transactions will solve this